crates/solidity/outputs/cargo/slang_solidity/src/generated/kinds.rs

```diff
@@ -775,6 +773,21 @@ pub enum TokenKind {
     YulYearsKeyword,
 }
 
+impl TokenKind {
+    pub fn is_trivia(&self) -> bool {
```

Comment:
I wonder if there is a limitation to using `match!()` macro here?

Reply:
It does not handle empty set (rare, but still) and the autoformatting is off, because it's inside a Rust macro and Rustfmt uses a badly looking heuristic:

```rust
        matches!(self, |Self::EndOfLine| Self::MultiLineComment
            | Self::MultiLineNatSpecComment
            | Self::SingleLineComment
            | Self::SingleLineNatSpecComment
            | Self::Whitespace)
```